projects/solid/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@maskito/solid",
+    "version": "0.0.1",
+    "description": "",
+    "license": "MIT",
+    "author": "",
+    "type": "module",
+    "module": "dist/index.js",
+    "scripts": {
+        "start": "jiti ./src/index.ts",
+        "build": "tsup"
```

Comment:
We use [Nx](https://nx.dev) tools to build all Maskito's libraries.
Let's use Nx for the new library too.